api/definitions.yaml

```diff
@@ -241,6 +241,28 @@ Service:
   - department_name
   - authorized_cidrs
   - authorized_recipients
+ServicePublic:
+  title: Service (public)
+  description: A Service tied to an user's subscription.
+  type: object
+  properties:
+    service_id:
+      $ref: "#/ServiceId"
+    service_name:
+      $ref: "#/ServiceName"
+    organization_name:
+      $ref: "#/OrganizationName"
+    department_name:
+      $ref: "#/DepartmentName"
+    version:
+      type: integer
+    id:
```

Comment:
the actual document ID is redundant as it is composed of `service_id` and `version` and should be used internally only